collect_app/src/main/AndroidManifest.xml

```diff
@@ -181,6 +181,10 @@ the specific language governing permissions and limitations under the License.
             android:name=".provider.InstanceProvider"
             android:authorities="org.odk.collect.android.provider.odk.instances"
             android:exported="true" />
+        <provider
+            android:name=".provider.ProjectsProvider"
+            android:authorities="org.odk.collect.android.provider.odk.projects"
+            android:exported="true" />
```

Comment:
We could use this new provider as an opportunity to introduce [permissions](https://developer.android.com/guide/topics/providers/content-provider-creating). In fact, we may want to do this for all providers with this release. On one hand, there isn't much sensitive exposed by any of these anymore. On the other, project URI, instance_name, geometry actually could be quite sensitive.

I would go with a single read/write provider-level permission. E.g. `org.odk.collect.android.provider.permission.READ_WRITE_PERMISSION`. Ideally it would be shared between all providers, I think.

Reply:
I think we should plan this as a follow-up issue, so we can discuss properly. Again, I've added an item to our catch-up.

Reply:
I agree it should be a separate issue.